Gifski/MainWindowController.swift

```diff
@@ -0,0 +1,175 @@
+import Cocoa
+import ProgressKit
+import DockProgress
+
+extension NSNib.Name {
+	static let mainWindowController = NSNib.Name("MainWindowController")
+}
+
+class MainWindowController: NSWindowController {
+
+	private var progressObserver: NSKeyValueObservation?
+
+	lazy var circularProgress = with(CircularProgressView(frame: CGRect(widthHeight: 160))) {
+		$0.foreground = .appTheme
+		$0.strokeWidth = 2
+		$0.percentLabelLayer.setAutomaticContentsScale()
+		$0.percentLabelLayer.implicitAnimations = false
+		$0.layer?.backgroundColor = .clear
+		$0.isHidden = true
+		$0.centerInWindow(window)
+	}
+
+	lazy var videoDropView = with(VideoDropView()) {
+		$0.frame = window!.contentView!.frame
+		$0.dropText = "Drop a Video to Convert to GIF"
+		$0.onComplete = { url in
+			self.convert(url.first!)
+		}
+	}
```

Comment:
Can you make these two `private`? I usually prefer to keep everything private unless it needs to be used elsewhere. That way I ensure nothing is used in places where I didn't explicitly intend it to be used.